spec/request/session/create_spec.rb

```diff
@@ -35,6 +35,30 @@
         expect { subject }.to(change { User.last.player_id })
       end
     end
+
+    context 'when logging in with facebook' do
+      let!(:facebook_params) { { access_token: '123456', format: :json } }
+
+      subject { post facebook_api_v1_users_path, params: facebook_params, as: :json }
```

Comment:
don't you have to mock the one signal request here?

Reply:
yes, this is done in the outer context